includes/Modules/Analytics.php

```diff
@@ -325,6 +325,23 @@ protected function print_amp_gtag() {
 			),
 		);
 
+		/**
+		 * Filters configuration options for the object that is passed to amp-analytics as a JSON component.
+		 *
+		 * @since 1.0.0
+		 *
+		 * @see https://developers.google.com/gtagjs/devguide/amp
+		 *
+		 * @param array $gtag_amp_opt gtag amp options.
+		 */
+		$config = apply_filters( 'googlesitekit_amp_gtag_opt', $gtag_amp_opt );
```

Comment:
Let's name this variable `$gtag_amp_opt_filtered` instead of `$config`, to clarify the association.